filters/apiusagemonitoring/spec.go

```diff
@@ -124,6 +125,9 @@ type apiUsageMonitoringSpec struct {
 	realmsTrackingMatcher *regexp.Regexp
 	unknownPath           *pathInfo
 	sometimes             rate.Sometimes
+
+	mu        sync.Mutex
+	filterMap map[string]*apiUsageMonitoringFilter
```

Comment:
This will have the usual problem of potentially growing unbounded because it is never cleaned up.

Reply:
Should be fine to keep it like that